src/layouts/InterfaceLayout/components/InterfaceNetworkModal/InterfaceNetworkModal.vue

```diff
@@ -31,11 +31,11 @@
           :key="key + index"
           class="content-block">
           <div class="network-title">
-            <img 
-              v-if="key === 'ROP' || key === 'RIN' || key === 'KOV'" 
+            <img
+              v-if="key === 'ROP' || key === 'RIN' || key === 'KOV' || key === 'ATH'"
```

Comment:
I think this should be a computed value so, we can use an array or something instead of having them hard coded.

Reply:
@SteveMieskoski actually, it can just be a boolean. let me update it.